### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by CrypticNeon45** [upptime/upptime#790] [Q&A]
Hello,

I hope this message finds you well. I'm writing to ask a favour. I recently supporting a GitHub repositories and I would be grateful if you could review them and leave star on it. Starring the repository helps to increase its visibility and makes it easier for others to find and use.

Here is the link to the repositories: 
↪https://github.com/mineiros-io/terramate

If you have a moment, I would really appreciate it if you could review and star the repositories if you find it useful.

Thank you very much.

Can you please star this repo. it is an open source project and we are trying to reach out as many people as we can

**Comment by AnandChowdhary** [comment]
Please don't spam
